research/integrations/fedora-ci/index.md

```diff
@@ -0,0 +1,338 @@
+---
+title: Packit as Fedora dist-git CI
+authors: mfocko
+---
+
+Related links:
+
+- [Epic](https://github.com/packit/packit-service/issues/2453)
+- [Research issue](https://github.com/packit/packit-service/issues/2462)
+
+## Current Fedora CI
+
+1. Koji scratch build
+1. Installability
+1. tmt (if plans are present)
+
+### Zuul as a parallel CI
+
+1. Builts SRPM
+1. Submits Koji scratch build
+1. Runs RPM linter
+1. Runs RPM inspect
+1. Tests install
+1. Runs tmt
+
+## Possible implementation
+
+### SRPM build
+
+Ideally kept identical. Packit should not be involved in any way that can affect
+the results of the SRPM build, i.e., just run `fedpkg srpm` and be done with it.
+
+This can be done within the sandbox as we do for non-scratch builds. However it
+might need some tweaking as Packit has specific worfklow.
+
+### Koji scratch build
+
+We are already submitting scratch builds for merged PRs, if configured. This
+would just extend the functionality to the non-merged PRs (with need to provide
+the SRPM).
+
+### RPM linting, inspect and installability test
+
+There are premade tmt plans that we can run.
+
+:::note Question
+
+Should these be run in parallel or sequentially and fail on first?
+
+:::
```

Comment:
Are some of these somehow dependant?

Reply:
From the excerpt here (linting, installability, tests), it feels like linting should not be a hard blocker. I've seen the installability tests fail (even if tests passed) just because of the network flakes and such, so…

In the sense of saving the resources, it definitely makes sense to make _full test suite_ dependent on the _installability_, but if it's flakey, it's just annoying.